RHDEVDOCS-7551: CQA assessment and fixes for the  Securing webhooks with event listener chapter. (#113107)
openshift/openshift-docs: modules/op-sample-eventlistener-resource.adoc

:_mod-docs-content-type: PROCEDURE
[id='op-sample-eventlistener-resource_{context}']
= Create a sample EventListener resource using secure HTTPS connection

[role="_abstract"]

> Create a sample EventListener resource with secure HTTPS connectivity using the pipelines-tutorial examples for hands-on learning and testing webhook security configurations.

Reviewer: ```suggestion
Create a sample `EventListener` resource with secure HTTPS connectivity using the pipelines-tutorial examples for hands-on learning and testing webhook security configurations.
```
A resource should be inside code blocks.

Author: Updated.

----
$ oc create route reencrypt --service=<svc_name> --cert=tls.crt --key=tls.key --ca-cert=ca.crt --hostname=<hostname>
----

.Verification

> * Verify that the EventListener is created and is available:

Reviewer: ```suggestion
* Verify that the `EventListener` resource is created and is available:
```
Suggestion to use a noun definer with blocks.

Author: Updated.

+
[source,terminal]
----

> oc get eventlistener

Reviewer: ```suggestion
$ oc get eventlistener
```

Author: Updated.

----
+

> The output shows the EventListener with `AVAILABLE` and `READY` columns indicating `True`.

Reviewer: ```suggestion
The output shows the `EventListener` resource with `AVAILABLE` and `READY` columns indicating `True`.
```

Author: Updated.

> * Verify that the EventListener service is created:

Reviewer: ```suggestion
* Verify that the `EventListener` service is created:
```

Author: Updated.

+
[source,terminal]
----

> oc get svc -l eventlistener=<eventlistener_name>

Reviewer: ```suggestion
$ oc get svc -l eventlistener=<eventlistener_name>
```

Author: Updated.

----
+
The output shows a service prefixed with `el-` and the port `8080/TCP` or `8443/TCP` for HTTPS.

* Verify that the other trigger resources are created:
+
[source,terminal]
----

> oc get trigger,triggertemplate,triggerbinding

Reviewer: ```suggestion
$ oc get trigger,triggertemplate,triggerbinding
```

Author: Updated.

----

* Verify that the secure route is accessible:
+
[source,terminal]
----

> curl -k https://<hostname>/

Reviewer: ```suggestion
$ curl -k https://<hostname>/
```

Author: Updated.

----
+

> The EventListener responds with a success message. The `-k` flag bypasses certificate verification for testing purposes.

Reviewer: ```suggestion
The Event Listener responds with a success message. The `-k` flag bypasses certificate verification for testing purposes.
```
This is a general comment for all occurrences of _EventListener_. If you are referring to the service or resource, it should be written as `EventListener`. However, if you are using it as a generic function or concept, I recommend using event listener (or Event listener, as appropriate based on context and style guidelines).

Author: Noted and updated.
